syncopy/io/load_spy_container.py

```diff
@@ -28,108 +26,108 @@
 def load(filename, tag=None, dataclass=None, checksum=False, mode="r+", out=None):
     """
     Load Syncopy data object(s) from disk
-    
+
     Either loads single files within or outside of '.spy'-containers or loads
-    multiple objects from a single '.spy'-container. Loading from containers can 
-    be further controlled by imposing restrictions on object class(es) (via 
-    `dataclass`) and file-name tag(s) (via `tag`). 
-    
+    multiple objects from a single '.spy'-container. Loading from containers can
+    be further controlled by imposing restrictions on object class(es) (via
+    `dataclass`) and file-name tag(s) (via `tag`).
+
     Parameters
     ----------
     filename : str
         Either path to Syncopy container folder (\*.spy, if omitted, the extension
         '.spy' will be appended) or name of data or metadata file. If `filename`
-        points to a container and no further specifications are provided, the 
```

Comment:
So it was not always only me haveing this white spaces all over the place :sweat_smile: